packages/sdk/poetry.lock

```diff
@@ -364,22 +364,20 @@ type = ["mypy (>=1.11.2)"]
 
 [[package]]
 name = "protobuf"
-version = "4.25.6"
+version = "6.30.0"
```

Comment:
This confirms that the shipping packages no longer restrict users from using protobuf v6. Only the private codegen dependencies have this limitation.